### PR TITLE
Fix rubocop issues

### PR DESCRIPTION
**Pull request by revolter**

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -107,6 +111,9 @@ Metrics/ParameterLists:
 Metrics/PerceivedComplexity:
   Max: 18
 
+Metrics/BlockLength:
+  Enabled: false
```
Comment (revolter) [inline .rubocop.yml]: @orta, Let me know if this is ok.

Comment (orta) [reply .rubocop.yml]: Very

**Comment by orta** [comment]
Nice work, good ruby skillllzzzzzz

**Comment by orta** [comment]
I added you as a contributor, no need to feel additional pressure to do anything, but at least now you can control your destiny if you need to

**Comment by revolter** [comment]
Whaaat? No, I'm a newbie ruby (hmm, that rhymes).

**Comment by revolter** [comment]
@orta, OMG thank you!

**Comment by orta** [comment]
You got this

**Pull request merged**